drivers/serial/Kconfig.cdns

```diff
@@ -0,0 +1,11 @@
+#Copyright Meta Platforms, Inc.and its affiliates.
```

Comment:
Please add year number to all the Copyright statements in this PR.

Reply:
Updated. Thanks.